quinn-proto/src/congestion.rs

```diff
@@ -65,6 +65,15 @@ pub trait Controller: Send + Sync {
     /// Number of ack-eliciting bytes that may be in flight
     fn window(&self) -> u64;
 
+    /// Update implementation-specific metrics.
```

Comment:
Maybe mention that this is used to populate qlog traces when they're enabled? Might be pretty mysterious otherwise.